sdk/storage/Azure.Storage.Files.DataLake/src/Models/PathProperties.cs

```diff
@@ -176,6 +176,12 @@ public class PathProperties
         /// </summary>
         public DateTimeOffset ExpiresOn { get; internal set; }
 
+        /// <summary>
+        /// Returns the name of the encryption scope used to encrypt the path contents and application metadata.
+        /// Note that the absence of this header implies use of the default account encryption scope.
```

Comment:
can we define encryption scope at account level ? should this be `use of default file system encryption scope` ?

Reply:
There is an account level encryption scope, but it isn't user-definable.  I'll fix this comment.

Reply:
Fixed.